### PR TITLE
Add a way to destroy plugin handle

### PR DESCRIPTION
**Pull request by bblaszkow06**

### CODE REVIEW

File: lib/plugin/videoroom.ex [hunk 1]
```diff
@@ -32,17 +32,30 @@ defmodule Janus.Plugin.VideoRoom do
...
 
-  @spec attach(Session.t(), Session.timeout_t()) ::
-          {:error, any} | {:ok, Session.plugin_handle_id()}
   @doc """
   Attaches to a videoroom plugin, creating a new handle
 
   Simple wrapper over `#{inspect(Session)}.session_attach/3`
   """
-  def attach(session, timeout \\ 5000) do
+  @spec new_handle(Session.t(), Session.timeout_t()) ::
```
Comment (Qizot) [inline lib/plugin/videoroom.ex:40]: Why not just use `attach` and `detach` names just like in janus?

Comment (bblaszkow06) [reply lib/plugin/videoroom.ex:40]: Do you really think the names are good? And obvious to anyone not used to Janus quirks?

Comment (Qizot) [reply lib/plugin/videoroom.ex:40]: In the end someone that uses janus have to know how it works and would expect to find attach and detach functionality based on janus documentation.

Comment (Qizot) [reply lib/plugin/videoroom.ex:40]: Nevermind, I mistook this repository for `elixir_janus`.

**Review by Qizot — APPROVED** [review]

**Pull request merged**